packages/Webkul/BookingProduct/src/Type/Booking.php

```diff
@@ -22,10 +22,10 @@
 
 class Booking extends Virtual
 {
-    /** @var bool 
-     * 
-     * do not allow booking products to be copied, it would be too complicated. 
-     * 
+    /** @var bool
+     *
+     * do not allow booking products to be copied, it would be too complicated.
+     *
      */
```

Comment:
Please set this doc beautifully, you can take reference from other properties also.

Reply:
how do I set the doc 'beautifully'?

Reply:
~~~php
/**
 * Do not allow booking products to be copied, it would be too complicated.
 *
 * @var bool
 */
~~~